app/javascript/hyacinth_v1/components/projects/publish_targets/PublishTargetEdit.js

```diff
@@ -1,36 +1,20 @@
 import React from 'react';
-import gql from 'graphql-tag';
 import { useQuery } from '@apollo/react-hooks';
 import { useParams } from 'react-router-dom';
+import { upperCase } from 'lodash';
```

Comment:
I don't think so, I just didn't know about the js built-in method. I will use that one instead.

Reply:
Don't worry about it! we'll do a review of lodash generally later on.

Reply:
For a plain string there's no difference, but lodash's upperCase also handles arrays of strings like this:

```
_.upperCase(["first string", "second string"])
// "FIRST STRING SECOND STRING"
```

Since it's an easy change to make, maybe good to swap it out now so we don't forget about it later?  I'd be happy to add a commit with the change if that's helpful.